docs/conf.py

```diff
@@ -376,11 +338,6 @@
 linkcheck_timeout = 180
 linkcheck_anchors = False
 
-# Add any extra paths that contain custom files (such as robots.txt or
-# .htaccess) here, relative to this directory. These files are copied
-# directly to the root of the documentation.
-html_extra_path = ["robots.txt"]
```

Comment:
Perhaps we should instead add robots.txt. Or open an issue to add one!

Reply:
This was moved above in the HTML section.